provision/ovh.go

```diff
@@ -75,8 +76,32 @@ func (o *OVHProvisioner) Status(id string) (*ProvisionedHost, error) {
 	}, nil
 }
 
+func (o *OVHProvisioner) lookupID(request HostDeleteRequest) (string, error) {
+	instances, err := o.client.ListInstance(context.Background())
+	if err != nil {
+		return "", err
+	}
+	for _, instance := range instances {
+		ip4, _ := ovhsdk.IPv4(&instance)
```

Comment:
It's non obvious why you didn't include error handling, so you could add it here, or add comment to say why.

Why?

Someone will only add it later, when they are "refactoring" or "improving" the code, so it's better to be intentional if we want to avoid that.

Reply:
roger that. Will add it with the next PR!